index.js

```diff
@@ -49,4 +49,45 @@ const elementReady = (selector, {
 	return Object.assign(promise, {stop});
 };
 
+elementReady.subscribe = (selector, callback, {
+	target = document,
+	stopOnDomReady = true,
+	timeout = Infinity
+} = {}) => {
+	const seenElements = new WeakSet();
+
+	let rafId;
+	let checkFrame = true;
+
+	const stop = () => {
+		cancelAnimationFrame(rafId);
+	};
+
+	if (timeout !== Infinity) {
+		setTimeout(stop, timeout);
+	}
+
+	(function check() {
+		if (checkFrame) {
+			const allElements = target.querySelectorAll(selector);
+
+			for (const element of allElements) {
+				if (seenElements.has(element)) {
+					continue;
+				}
+
+				callback(element);
```

Comment:
Can you pass `undefined` instead of `null` (which `querySelectorAll` returns). To be consistent with our other usage of `undefined`.